cloudbuild_public.yaml

```diff
@@ -42,20 +42,24 @@ steps:
   - |
     # Only push to Docker Hub from master
     [ "$BRANCH_NAME" != "master" ] && exit 0
-    docker pull us.gcr.io/$PROJECT_ID/$REPO_NAME:$COMMIT_SHA 
+    docker pull $$SENTRY_IMAGE 
```

Comment:
Please make this `$$CDC_IMAGE` as `SENTRY_IMAGE` has a special meaning in on-premise and it will interfere.